inference-engine/tests_deprecated/functional/ie_tests/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ add_library(${TARGET_NAME} STATIC ${TEST_INCLUDE} ${TEST_SRC})
 list(APPEND EXPORT_DEPENDENCIES
         funcTestUtils
         ieTestHelpers
+        ieTestHelpers_s
```

Comment:
does it looks strange that the same binary is linked against both static and non-static versions?

Reply:
I've thoght that this variable was used for `ie_developer_export_targets` only. Fixed.

Reply:
BTW, `ieTestHelpers_s` is a part of deprecated tests infra, why do you need to use it?
CC @iefode what do you think?

Reply:
As well as `ieTestHelpers`. I don't see reasons to export only one of them.